frontend/src/pages/Player/PlayerHook/PlayerState.ts

```diff
@@ -838,7 +838,7 @@ const processSessionMetadata = (
 			sessionIndex: number
 		})[] = []
 
-		sessionIntervals.forEach((interval, sessionIndex) => {
+		s.sessionIntervals.forEach((interval, sessionIndex) => {
```

Comment:
i believe this is no different, but better to use the `s.sessionIntervals` object here for consistency